recipes/ca-certificates/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2016.2.28" %}
+
+package:
+  name: ca-certificates
+  version: {{ version }}
+
+# This does not have a source because it pulls these from certifi at present.
+
+build:
+  number: 0
+  skip: true  # [win or not py35]
```

Comment:
You do need this for less-native programs, like curl.  Otherwise, Windows has its own way of doing things, and larger things that we are rebundling are going to include their own anyway.  This would be nice to have for Curl on win, though.